grpc/src/main/java/com/linecorp/armeria/server/grpc/HandlerRegistry.java

```diff
@@ -179,12 +196,17 @@ HandlerRegistry build() {
             final ImmutableMap.Builder<MethodDescriptor<?, ?>, String> bareMethodNames = ImmutableMap.builder();
             final ImmutableMap.Builder<ServerMethodDefinition<?, ?>, List<DecoratorAndOrder>> decorators =
```

Comment:
optional; it might be easier on other developers if the variable name is updated as well

```suggestion
            final ImmutableMap.Builder<ServerMethodDefinition<?, ?>, List<DecoratorAndOrder>> annotationDecorators =
```